prow/cmd/mkbuild-cluster/main.go

```diff
@@ -199,14 +227,25 @@ func describeCluster(o options) (*describe, error) {
 	if d.Endpoint == "" {
 		return nil, errors.New("empty endpoint")
 	}
-	if len(d.Auth.ClusterCACertificate) == 0 {
-		return nil, errors.New("empty clusterCaCertificate")
-	}
 
-	if len(d.Auth.ClientKey) == 0 {
+	// Only generate new certificate if necessary
+	if o.getClientCert && (len(d.Auth.ClusterCACertificate) == 0 || len(d.Auth.ClientKey) == 0 || len(d.Auth.ClientCertificate) == 0) {
+		clientset, err := createDefaultClient(ctx)
+		if err != nil {
+			return nil, fmt.Errorf("create kube client: %v", err)
```

Comment:
This will output `create kube client: create kube clientset from config: WHATEVER` which is possibly excessively verbose.

Reply:
Is the preferred solution to reduce verbosity to just return `err` in `createDefaultClient`?

Reply:
IMO no, but repeating kube 50 times also isn't useful, `create client: get config: whatever.yaml does not exist` seems perfectly comprehensible

Reply:
Thanks for the clarity. I have shorten all errors as you suggested.